### PR TITLE
translate “工作 996，生病 ICU” to english version.

### PR DESCRIPTION
**Pull request by dawangraoming**
translate "工作 996，生病 ICU" to english version:

996 working with you, will be live in ICU.

### CODE REVIEW

**Comment by wangexperience** [comment]
> translate "工作 996，生病 ICU" to english version:
> 
> 996 working with you, will be live in ICU.

啊哈哈，翻译的没毛病吖

**Comment by seven7goddess** [comment]
interesting

**Comment by Crisimple** [comment]
🔥了🔥了

**Comment by reilyxu** [comment]
996 working, ICU waiting

**Review by 996icu — CHANGES_REQUESTED** [review]
It's would be better to put this translation at the same place of "工作 996，生病 ICU" in Chinese version.

**Comment by Opdoop** [comment]
> It's would be better to put this translation at the same place of "工作 996，生病 ICU" in Chinese version.

How about "Working 996, way to ICU" ?

**Comment by piaoyezju** [comment]
working 996 , dying icu

**Comment by guikarist** [comment]
@dawangraoming 

I think this one is better and free of grammatical errors:

_**996 working, ICU requiring.**_

**Comment by sincerefly** [comment]
week week 996, day day ICU

File: en_US.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 ===
 
 ## What is "996"?
+996 working with you, will be live in ICU.
```
Comment (liufuyang) [inline en_US.md]: how about:
`Working like 996, soon sick at a ICU`

Comment (guikarist) [reply en_US.md]: Honestly, too complicated.

Comment (liufuyang) [reply en_US.md]: Yeah I feel the above `996 working, ICU waiting` sounds good as well 😆

**Comment by johnmahui1** [comment]
running 996, to be ICU

File: en_US.md (hunk 1 above)
Comment (cokabug) [reply en_US.md]: 996 working, ICU waiting +1

Comment (jokerYellow) [reply en_US.md]: 96 working, ICU waiting +1. Great translating

**Comment by ohmydone** [comment]
996 lead you to ICU

**Comment by yahkun** [comment]
996 working, ICU waiting +1

**Comment by unixzii** [comment]
A 996 a week keeps the ICU `!away`;

**Comment by dawangraoming** [comment]
@996icu 将这段单句放到与中文版本相同位置是一个很好的建议，不过因为英文版本经过重组与排版后，在结构上已经和中文版本有较大的差别了，我建议放在`What is "996"`介绍部分下会更合理一些。

**Comment by dawangraoming** [comment]
很感谢大家热情参与到这个PR中。

@reilyxu 翻译的`996 working, ICU waiting`受到很多用户的点赞，简短且押韵，我也很喜欢这条翻译。
当然每个人心中都有一个自己的哈姆雷特，所以我在知乎上提了一个问题，欢迎大家在这里提出自己喜欢的翻译。

[“工作996 生病ICU”有哪些更好的翻译？](https://www.zhihu.com/question/317765123)

**Comment by kimown** [comment]
996 working, ICU waiting  +1

**Comment by guikarist** [comment]
However, “ICU waiting” has grammatical errors. “wait” is intransitive, you cannot say “wait ICU” but “wait for ICU”.

Unless you regard “ICU” as a adverbial just like “996” in the first part of the sentence. So “ICU living” or “ICU requiring” is more grammatically reasonable than “ICU waiting”.

**Comment by hqhzz1993** [comment]
If you hate somebody ,give 996 to him/her.

**Comment by HotManTH** [comment]
@Kris Wu，进来学说唱

> 996 working, ICU waiting

**Comment by HotManTH** [comment]
@KrisWu 进来学说唱。

> 996 working, ICU waiting

**Comment by mypridelife** [comment]
翻译不是乱译，不然就开花
前排留名，火了叫我

**Comment by leaderliang** [comment]
interesting

**Review by 996icu — APPROVED** [review]

**Comment by yyywyr** [comment]
go to icu,from 996

**Comment by troj-iii** [comment]
> week week 996, day day ICU

外国人能懂的英文，既有中文特色，also has english meaning.

**Comment by PengZhao12** [comment]
翻译翻译，什么叫惊喜，什么他妈的叫惊喜

**Comment by Myluckygirl** [comment]
996 working, ICU waitin.

**Comment by TreeZhiyuan** [comment]
working with 996， dead in ICU

**Comment by emcvhkr** [comment]
996 a Week, ICU be Quick!

**Comment by ValueYouth** [comment]
其实chinglish也是可以被很多外国有人理解的，就像他们断断续续的中文，我们也可以理解一样

**Comment by wytwww** [comment]
hahaha

**Comment by iocout** [comment]
today 996 ,tomorry ICU  ¿

**Comment by iocout** [comment]
today 996  ，tomorrow ICU ¿
  haha

**Pull request merged**